apps/portals/e2e/explore.spec.ts

```diff
@@ -0,0 +1,362 @@
+        for (let i = 0; i < nAvailableFacets; i++) {
+          const buttonText = await availableFacetButtons.nth(i).textContent()
+          buttonText && availableFacetNames.push(buttonText)
+        }
+        return availableFacetNames
+      })
+
+    for (let i = 0; i < nChartCells; i++) {
+      await test.step(`confirm chart ${i}`, async () => {
+        const chartCell = chartCells.nth(i)
+        await expect(chartCell).toBeVisible()
+
+        await test.step('confirm chart title', async () => {
+          const chartTitle = chartCell.locator('.FacetNavPanel__title')
+          const chartTitleText = await chartTitle.textContent()
+          expect(availableFacetNames).toContain(chartTitleText)
+          await expectButtons(chartTitle, [
+            'Filter by specific facet',
+            'Expand to large graph',
+            'Hide graph under Show More',
+          ])
+        })
+
+        await test.step('confirm chart body', async () => {
+          const chartBody = chartCell.locator('.FacetNavPanel__body')
+          await expect(chartBody.locator('.js-plotly-plot')).toBeVisible()
+          await expect(
+            chartBody.locator('.FacetNavPanel__body__legend'),
+          ).toBeVisible()
+        })
+      })
+    }
+
+    await test.step('confirm view all charts button', async () => {
+      const button = chartsPanel.getByRole('button', {
+        name: 'View All Charts',
+      })
+
+      const nFacetControlLabels = await getFacetFilterControls(page)
+        .locator('.FacetFilterHeader__label')
+        .count()
+      if (nFacetControlLabels <= 2) {
+        await expect(button).not.toBeVisible()
+      }
+      if (await button.isVisible()) {
+        expect(nFacetControlLabels).toBeGreaterThan(2)
+      }
+    })
+  })
+}
+
+const expectCharts = async (
+  page: Page,
+  exploreTab: string,
+  isTableTab: boolean,
+) => {
+  await test.step('confirm charts', async () => {
+    await expectTopLevelControls(page, exploreTab, isTableTab)
+    await expectFacetControls(page)
+    await expectFacetCharts(page)
+  })
+}
+
+const expectSynapseProfileHref = (href: string) => {
+  expect(href).toMatch(/^https:\/\/www.synapse.org\/#!Profile:\d{6,}$/)
+}
+
+const expectPeopleCards = async (page: Page) => {
+  await test.step('confirm people cards', async () => {
+    const cards = await test.step('expect people card list', async () => {
+      const cards = page.locator('.SRC-userCard')
+      await expect.poll(() => cards.count()).toBeGreaterThanOrEqual(1)
+      return cards
+    })
+
+    await test.step('expect people cards have subcomponents', async () => {
+      const nCards = await cards.count()
+      for (let i = 0; i < nCards; i++) {
+        await test.step(`confirm card ${i}`, async () => {
+          const card = cards.nth(i)
+
+          const isSynapseUser = await card
+            .filter({ has: cards.getByText('synapse') })
+            .isVisible()
+
+          if (isSynapseUser) {
```

Comment:
Some people in the Elite Portal do not have Synapse user info displayed, so conditionally check for Synapse-specific card info